baker/Deployer.ts

```diff
@@ -0,0 +1,374 @@
+import * as fs from "fs-extra"
+import * as prompts from "prompts"
+import ProgressBar = require("progress")
+import { execWrapper } from "../db/execWrapper"
+import { spawn } from "child_process"
+import simpleGit, { SimpleGit } from "simple-git"
+import { WriteStream } from "tty"
+import { ProgressStream } from "./ProgressStream"
+
+const TEMP_DEPLOY_SCRIPT_SUFFIX = `.tempDeployScript.sh`
+
+enum DeployTarget {
```

Comment:
should probably name this something else (`StagingDeployTarget`?) to indicate that it doesn't include `live`.

Also we could maybe benefit by having that enum in its own file, or at least exporting it? https://github.com/owid/owid-grapher/blob/master/deploy/liveCommit.ts also uses it.

Reply:
Good call. Done and done.